ecosystem/typescript/sdk_v2/src/types/index.ts

```diff
@@ -50,6 +50,7 @@ export enum TransactionAuthenticatorVariant {
   TransactionAuthenticatorEd25519 = 0,
   TransactionAuthenticatorMultiEd25519 = 1,
   TransactionAuthenticatorMultiAgent = 2,
+  TransactionAuthenticatorSecp256k1 = 3,
```

Comment:
were gonna have conflicts here https://github.com/aptos-labs/aptos-core/pull/10387/files#diff-dfcab1d73be97cd928651c6d57b8561b1b8e9ce470fd47ad8ac52be280647240R68

Reply:
Ahh will resolve the conflict afterward